pylua/timer.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+# timer works with milliseconds
+
+import threading
+
+
+def msec_to_sec(milliseconds):
+    milliseconds_in_second = 1000.0
+    return milliseconds/milliseconds_in_second
+
+
+def sec_to_msec(seconds):
+    milliseconds_in_second = 1000.0
+    return seconds*milliseconds_in_second
+
+
+class PyTimer(object):
+
+    def __init__(self, task, interval):
+        self._task = task
+        self._interval = msec_to_sec(interval)
+        self._timer = None
+
+    def start(self):
+        self.create()
+        self._timer.start()
+
+    def create(self):
+        self._timer = threading.Timer(self._interval, self._task)
```

Comment:
@malirod I use create because I can not start timer which finised work again  For this reason need to recreate timer.